_posts/2018-10-21-nabla-k8s.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+title: Nabla on Kubernetes!
+---
+
+A while back, we released a demo of the nabla containers runtime, `runnc` (v0.1). Most recently, we have improved our runtime in several aspects such as better start-up times and memory density. In addition to that (and also the focus of this blogpost), we have implemented Nabla support for kubernetes! In this blogpost, we will show a simple local setup of kubernetes using the nabla containers runtime, `runnc` (v0.2), through [containerd](https://github.com/containerd/containerd)  via the [CRI untrusted workload cri plugin and pod annotation](https://github.com/containerd/cri/blob/master/pkg/annotations/annotations.go).
+
+**Announcement: We will presenting Nabla Containers at Kubecon China 2018, and a joint presentation with Katacontainers at Kubecon Seattle 2018, so do come by and chat with us!**
```

Comment:
Should this be in this article or should we just put up an additional article with the announcement on?  It's just that the announcement will be out of date in a month or so :)